rust/Cargo.toml

```diff
@@ -22,18 +22,28 @@ bytes = "0.5.3"
 log = "0"
 regex = "1"
 chrono = "0"
-rusoto_core = "0.43"
-rusoto_s3 = "0.43"
+
+# Azure
+azure_core = { git = "https://github.com/Azure/azure-sdk-for-rust", optional = true}
```

Comment:
given that the rust sdk is in active development, it's more likely for it to break our azure build if we don't pin to a specific commit.

Reply:
Makes sense - done.